compclub/website/urls.py

```diff
@@ -1,10 +1,14 @@
-from django.urls import include, path
+from django.conf.urls import url
+from django.contrib.auth import views as auth_views
+from django.urls import path
 
 from . import views
 
 urlpatterns = [
+    url(r'^login/$', auth_views.login, {'template_name': 'website/login.html'}, name='login'),
+    url(r'^logout/$', auth_views.logout, {'next_page': '/'}, name='logout'),
```

Comment:
While this does work, let's use namespacing instead of raw URL strings for redirecting. This will make things much easier to change. To do this:
- On line 6, add `app_name = "website"`. This declares that the namespace for this URL file is "website".
- Change `'/'` to `'website:index'`